pyproject.toml

```diff
@@ -28,6 +28,7 @@ filterwarnings = [
     "error::modal.exception.DeprecationError",
     "ignore::DeprecationWarning:pytest.*:",
     "module:Jupyter is migrating its paths to use standard platformdirs:DeprecationWarning",
+    "ignore:.*pkg_resources.*:DeprecationWarning::",
```

Comment:
unrelated but needed on my newest dev box

Reply:
I got the same today.